pages.zh/linux/debootstrap.md

```diff
@@ -0,0 +1,20 @@
+# debootstrap
+
+> 创建一个基本的Debian系统。
+> 更多信息：<https://wiki.debian.org/Debootstrap>。
```

Comment:
@sandylaw per #5240 we've decided to use the English period at the end of the link sentence, because the link itself is English and the punctuation surrounding in (`<` and `>`) are halfwidth, like `.`.   All other sentences should use the Chinese period, though, like we've been doing.
```suggestion
> 更多信息：<https://wiki.debian.org/Debootstrap>.
```